### DISCUSSION TITLE
【灌水】“Ciallo～(∠・ω< )⌒☆”怎么打出来

### DISCUSSION THREAD

**Discussion by opqphp** [share121/inter-knot#4973] [其他]
Ciallo～(∠・ω< )⌒★Ciallo～(∠・ω< )⌒★Ciallo～(∠・ω< )⌒☆Ciallo～(∠・ω< )⌒☆另外sen ren* ban ka是什么

**Comment by Uknown-wang** [comment]
柚子厨蒸鹅心~~~

**Reply by opqphp** [reply]
![illust_117794406_20240819_202903.png](https://github.com/user-attachments/assets/f5dde945-6417-4d30-969a-23deae54879c)

……

**Comment by Mollsity** [comment]
天生邪恶的柚子厨，我这就亲手😡

**Reply by opqphp** [reply]
![illust_117794406_20240819_202903.png](https://github.com/user-attachments/assets/62a3eae5-3c19-4dfa-94b4-955d04fcef82)
